### PR TITLE
Add nodeSelector and tolerations field

### PR DESCRIPTION
**Pull request by kmdkuk**
#144 
Signed-off-by: kouki <kouworld0123@gmail.com>

### CODE REVIEW

File: controllers/runnerpool_controller_test.go [hunk 1]
```diff
@@ -478,6 +487,15 @@ var _ = Describe("RunnerPool reconciler", func() {
 					"Name": Equal(secretName),
 				}),
 			}),
+			"NodeSelector": MatchAllKeys(Keys{
```
Comment (masa213f) [inline controllers/runnerpool_controller_test.go:490]: Nice test!
Add to this, could you confirm that the NodeSelector and the Tolerations are not added in the minimal case?

**Review by masa213f — CHANGES_REQUESTED** [review]
minimal

File: controllers/runnerpool_controller_test.go (hunk 1 above)
Comment (kmdkuk) [reply controllers/runnerpool_controller_test.go:492]: @masa213f Thank you for the review.
I have added a test.
Please review again.

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**